print/components/story/StoryDay.vue

```diff
@@ -0,0 +1,90 @@
+<template>
+  <div class="tw-mb-4">
+    <div class="tw-text-2xl tw-mb-2 tw-break-after-avoid">
+      <h1>{{ $tc('entity.day.name') }} {{ day.number }} ({{ dateLong(day.start) }})</h1>
+    </div>
+
+    <template v-if="entriesWithStory.length">
+      <template v-for="{ scheduleEntry, storyChapters } in entriesWithStory">
+        <div v-for="chapter in storyChapters" :key="chapter._meta.uri" class="tw-mb-3">
+          <h4 class="tw-text-lg tw-font-bold tw-break-after-avoid">
+            <span class="d-inline-flex align-center">
+              <span>{{ scheduleEntry.number }}</span>
+              <category-label
+                :category="scheduleEntry.activity().category()"
+                class="tw-ml-1"
+              />
```

Comment:
Why did you use CategoryLabel instead of CategoryChip?

Reply:
By use you mean why this component is names "CategoryLabel", right? There's no `CategoryChip` component to use in the print portion.

Not changed in this PR, coming from earlier works from @carlobeltrame. But can rename the component from `CategoryLabel` to `CategoryChip` if this is a better naming.

Reply:
No, I mean the component used in Frontend Story Context `CategoryChip.vue`

Reply:
I created a separate one because I am of the opinion that PDF and Web are two different mediums which do not share the same interaction model. The category chip in frontend might be extended with a dropdown to change the category in the future, or it might have event handlers, hover styles, aria labels or title texts which are not relevant in a pdf. If we ever decide to go for a more 3d look of the frontend components, or add e.g. a drop shadow etc., this should in no way affect the pdfs. Logic that should be shared (e.g. an algorithm which decides whether the font color should be black or white, although even that rule might be different in print) can live in js files in the commons.
Maybe in this simple case, currently, they could be unified. But in the general case, these components are not supposed to be the same if you ask me. Otherwise the frontend components get unnecessarily complex if they also have to handle print styles.

Reply:
Okay. But I would like the styling to be the same as in the frontend. Then we should adapt that in a next PR.

Reply:
> But in the general case, these components are not supposed to be the same if you ask me

I agree to this. Currently we share translations and narrowly scoped helper functions. I wouldn't necessarily extend this to Vue-Components, unless there's a very compelling argument to do so.